app/controllers/alerts_list_controller.rb

```diff
@@ -23,8 +23,7 @@ def class_icons
       'ManageIQ::Providers::Kubernetes::ContainerManager::ContainerNode',
       'ManageIQ::Providers::Openshift::ContainerManager',
     ].each do |klass|
-      fileicon = klass.constantize.decorate.fileicon
-      res[klass] = ActionController::Base.helpers.image_path(fileicon)
+      res[klass] = klass.constantize.decorate.fonticon
```

Comment:
Random performance thing (not important to do though).  Can we make this a constant so we aren't generating it on every action?

Can be done in a separate PR, just something I wanted to comment on.

Reply:
Sure can do it later